lib/rules/i18n-no-reference.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+const { I18N_METHODS } = require('../i18n.js');
+
+function isReferenceNode(node) {
+  if (node.type === 'TemplateLiteral') return false;
+
+  return node.type !== 'Literal' || typeof node.value !== 'string'
+}
+
+function extractI18nMethodAndArgs(node, methodName) {
+  let result = [];
+
+  if (node.type === 'CallExpression') {
+    const { callee } = node;
+    if (callee.type === 'MemberExpression') {
+      const { object, property } = callee;
+      if (object.name === 'i18n') {
+        if (!methodName || (methodName !== property.name)) {
```

Comment:
I think that second check is wrong it should be methodName === property.name. I never actually pass a second arg so probably could just remove it... 😆

Reply:
Fixed.